setup.cfg

```diff
@@ -117,6 +117,8 @@ style =
     # pyupgrade 2.8+ required for --py39-plus flag
     pyupgrade>=2.8,<4
 tests =
+    # hydra-core 1.1.0+ required for recursive instantiation
```

Comment:
Are we using recursive instantiation anywhere?

Reply:
Not currently but a user is going to want this. I'll add it to the test config. Recursive instantiation allows you to define the callbacks, logger, augmentations inside the config file itself.